docs/rules/RedundantSyntax.md

```diff
@@ -31,7 +31,5 @@ Example:
 + println("Bar")
 
 - println(raw"Baz")
-+ println("Foo")
-
-println(raw"Foo\nBar")
```

Comment:
The intent of that line was to show that this wouldn't get rewritten. Did you remove it on purpose?

Reply:
Yes, that was my intention as it broke the pattern of the preceding three examples (or well, that's how I read it at least). But perhaps it's a good idea to keep it but with a comment saying that it's not rewritten as `raw` is indeed used.